Updated footer content links. closes vets.gov-team#678

webinista opened the pull request:
- Removed .usa-social-links CSS.
- Renamed va-list-linkgroup* to va-footer-linkgroup.
- Removed footer-logo block. Unnecessary.
- Switched footer to a flexbox layout.
- Moved disclaimer to bottom of footer.

webinista committed:
Updated footer content links. closes vets.gov-team#678

- Removed .usa-social-links CSS.
- Renamed va-list-linkgroup* to va-footer-linkgroup.
- Removed footer-logo block. Unnecessary.
- Switched footer to a flexbox layout.
- Moved disclaimer to bottom of footer.

webinista committed:
Updated footer content links. closes vets.gov-team#678

- Removed .usa-social-links CSS.
- Renamed va-list-linkgroup* to va-footer-linkgroup.
- Removed footer-logo block. Unnecessary.
- Switched footer to a flexbox layout.
- Moved disclaimer to bottom of footer.

webinista committed:
Merge branch 'department-of-veterans-affairs/vets.gov-team#678' of github.com:department-of-veterans-affairs/vets-website into department-of-veterans-affairs/vets.gov-team#678

webinista commented:
Screen shots of what this PR looks like. @emilyville please weigh in on this.

![480w](https://cloud.githubusercontent.com/assets/354277/21869291/d1ec6ee8-d80b-11e6-8c6a-53dd200eef80.png)
![320w](https://cloud.githubusercontent.com/assets/354277/21869292/d1ec5408-d80b-11e6-8d5f-db7d2d2ab4e3.png)
![desktop](https://cloud.githubusercontent.com/assets/354277/21869293/d1ef7ad4-d80b-11e6-9dbd-ede385e843b3.png)

webinista committed:
Simplifying media queries.

webinista committed:
Using full URLs instead of relative URLs so the tests pass

jkassemi committed:
Merge remote-tracking branch 'origin/master' into department-of-veterans-affairs/vets.gov-team#678

jkassemi committed:
Merge remote-tracking branch 'origin/master' into department-of-veterans-affairs/vets.gov-team#678

jkassemi commented:
Jenkins failure due to same issue as https://github.com/department-of-veterans-affairs/vets-website/pull/4703. Merged origin/master to incorporate the fix. Part of investigation here https://github.com/department-of-veterans-affairs/devops/issues/1352

webinista committed:
Merge branch 'master' into department-of-veterans-affairs/vets.gov-team#678

webinista committed:
Merge branch 'department-of-veterans-affairs/vets.gov-team#678' of github.com:department-of-veterans-affairs/vets-website into department-of-veterans-affairs/vets.gov-team#678

webinista committed:
Merge branch 'master' into department-of-veterans-affairs/vets.gov-team#678

webinista commented:
Renamed branch for this PR and moved to #4734 to work around a [Jenkins bug](https://github.com/department-of-veterans-affairs/vets-website/pull/4703#issuecomment-273310276)
